cypress/support/pages/dashboard/pageDashboard.js

```diff
@@ -5,7 +5,8 @@ class Dashboard {
 	static #languageMenu = "#language-menu";
 	static #selectedLanguage = "#selected-language";
 	static #listOfAllLanguages = "#available-languages";
-	static #germanLanguage = '[data-testid="available-language-de"]';
+	static #germanLanguageDevSel = '[data-testid="selected-language-de"]';
```

Comment:
What is DevSel suffix here? 
Is it then 'germanLanguageOptionOnDev' , 'germanLanguageOptionOnStaging'.

Reply:
done